### PR TITLE
WD-7763 - redesign /managed page

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Redesign [managed services page](https://ubuntu-com-13445.demos.haus/managed) according to [figma](https://www.figma.com/file/Qj0DByXnYYUn8BMtbB2OiA/24.04-Managed?type=design&node-id=0-1&mode=design&t=vHKZ7F3He1R87Ah5-0) and [copy doc](https://docs.google.com/document/d/19GoMudyrd9Zt1e7kfj-koL-Lt0RGYzWHRPuuKYUPvqw/edit)

## QA

- [demo link](https://ubuntu-com-13445.demos.haus/managed)
- [copy doc](https://docs.google.com/document/d/19GoMudyrd9Zt1e7kfj-koL-Lt0RGYzWHRPuuKYUPvqw/edit)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to the design and copy doc

## Issue / Card
[WD-7763](https://warthogs.atlassian.net/browse/WD-7763)

Fixes #

## Screenshots

[WD-7763]: https://warthogs.atlassian.net/browse/WD-7763?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
Review in figma https://www.figma.com/file/Qj0DByXnYYUn8BMtbB2OiA/24.04-Managed?type=design&node-id=135%3A2585&mode=dev

**Comment by akbarkz** [comment]
@lyubomir-popov Looks like you forgot adding `Design+1` label after you pushed the changes

File: templates/managed/index.html [hunk 1]
```diff
@@ -2,130 +2,151 @@
...
+      </div>
+      <div class="col-medium-3">
+        <div class="p-section--shallow">
+          <p>Gain the space to focus on your core business by outsourcing the management of your operations with
+            Canonical's Managed IT services. Whatever the size of your company, our engineers can deploy and manage your
+            environments efficiently and securely.
+          </p>
+        </div>
+        <hr>
+        <a class="p-button--positive" href="/managed#get-in-touch">Ask a Managed Solutions expert</a>
```
Comment (anthonydillon) [inline templates/managed/index.html:29]: This doesn't do anything. It's missing the class to invoke the interactive form

File: templates/managed/index.html [hunk 2]
```diff
@@ -134,455 +155,421 @@ <h1>Managed IT Services</h1>
...
-      hi_def=True,
-      loading="lazy"
-      ) | safe
-      }}
-    </div>
+<section class="u-fixed-width">
+  <hr class="p-rule">
+  <div class="p-strip is-deep">
+    <h2>
+      Case study: <a href="https://ubuntu.com/engage/kubernetes-openstack-case-study-firmus">Firmus builds a sustainable
```
Comment (anthonydillon) [inline templates/managed/index.html:165]: This link should be relative

File: templates/managed/index.html [hunk 3]
```diff
@@ -134,455 +155,421 @@ <h1>Managed IT Services</h1>
...
-      url="https://assets.ubuntu.com/v1/b0af9ede-UA_24-7_Support.svg",
-      alt="",
-      width="162",
-      height="162",
-      hi_def=True,
-      loading="lazy"
-      ) | safe
-      }}
+      <hr />
+      <a class="p-button--positive" href="/managed#get-in-touch">Tell us what you need</a>
```
Comment (anthonydillon) [inline templates/managed/index.html:188]: This doesn't do anything. It's missing the class to invoke the interactive form

File: templates/managed/index.html [hunk 4]
```diff
@@ -134,455 +155,421 @@ <h1>Managed IT Services</h1>
...
-              height="42",
-              hi_def=True,
-              loading="lazy",
-              ) | safe
-            }}
-          </div>
-          <hr />
-          <p>Landscape provides a dashboard and API for a wide range of administrative functions that encompass inventory, automation, hardening, compliance, reporting, and software distribution</p>
-          <p><a class="p-button--positive" href="/landscape/managed">Managed Landscape</a></p>
+<!-- Keep hidden until isometric illustrations arrive -->
```
Comment (anthonydillon) [inline templates/managed/index.html:394]: I think it's better to remove commented out code as it causes mess within the code base. We can use source control to recover the removed mark up.

File: templates/managed/index.html [hunk 5]
```diff
@@ -134,455 +155,421 @@ <h1>Managed IT Services</h1>
...
+              <td>
+                <h3 class="p-heading--5">Admin access</h3>
+              </td>
+              <td>Required</td>
+              <td>Not required</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
```
Comment (anthonydillon) [inline templates/managed/index.html:504]: Missing closing div

File: templates/managed/index.html [hunk 6]
```diff
@@ -134,455 +155,421 @@ <h1>Managed IT Services</h1>
...
+          </ul>
         </div>
       </div>
-      <h3 class="p-heading--4">
-        <a href="/engage/beat-disruption-wp">Beat Disruption: How to adapt your IT strategy for changing markets&nbsp;&rsaquo;</a>
-      </h3>
     </div>
   </div>
 </section>
-<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/managed-services.html" data-form-id="4053" data-lp-id="" data-return-url="https://ubuntu.com/managed/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html"> </div>
```
Comment (anthonydillon) [inline templates/managed/index.html:586]: We need this otherwise the `#get-in-tough` form will not work.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Looks nice @akbarkz, a few comments in the code.

File: templates/managed/index.html (hunk 2 above)
Comment (akbarkz) [reply templates/managed/index.html:160]: Sorry, I didn't understand this comment, can you please elaborate more?

File: templates/managed/index.html [hunk 7]
```diff
@@ -154,11 +157,11 @@ <h1>Managed IT Services you can rely on</h1>
 </section>
 
 <section class="u-fixed-width">
-  <hr class="p-rule" />
+  <hr class="p-rule">
```
Comment (akbarkz) [inline templates/managed/index.html]: @lyubomir-popov according to our convention all HTML tags should be self-closed.

File: templates/managed/index.html (hunk 2 above)
Comment (anthonydillon) [reply templates/managed/index.html:160]: It should not contain that domain and be relative so local and demo navigation remains on the same platform.

File: templates/managed/index.html [hunk 8]
```diff
@@ -134,455 +150,402 @@ <h1>Managed IT Services</h1>
...
+        Managed IT services are a solution that allows you to harness the power of open source without the hassle of
+        maintenance.
+      </p>
+      <p>
+        Canonical’s Managed Solutions is a service that identifies, deploys, and manages open source ecosystems for you
+        at predictable costs. This way, you can access our market-leading open source expertise and focus more of your
+        resources on your core business.
+      </p>
+      <hr />
+      <a class="p-button--positive js-invoke-modal" href="/managed#get-in-touch">Tell us what you need</a>
```
Comment (anthonydillon) [inline templates/managed/index.html:183]: The href should go to a static contract form as a fall back for no-js

File: templates/managed/index.html [hunk 9]
```diff
@@ -2,130 +2,146 @@
...
+<section class="u-no-margin--bottom">
+  <div class="p-section--shallow">
+    <div class="row">
+      <div class="col-medium-6 col-9 col-start-large-4">
+        <h1>Managed IT Services you can rely on</h1>
+        <p>Gain the space to focus on your core business by outsourcing the management of your operations with
+          Canonical's Managed IT services. Whatever the size of your company, our engineers can deploy and manage your
+          environments efficiently and securely.</p>
+        <hr />
+        <a class="p-button--positive js-invoke-modal" href="/managed#get-in-touch">Ask a Managed Solutions expert</a>
```
Comment (anthonydillon) [inline templates/managed/index.html:24]: Same here

**Review by anthonydillon — APPROVED** [review]
Some small requests before merging

File: templates/managed/index.html (hunk 8 above)
Comment (akbarkz) [reply templates/managed/index.html]: Good point, thanks 👍

**Pull request merged**